data/data/bootstrap/systemd/units/manifests.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Bootstrap an OpenShift cluster
+Wants=bootkube.service
+After=bootkube.service
+ConditionPathExists=!/opt/openshift/.manifests.done
+
+[Service]
+WorkingDirectory=/opt/openshift/manifests
```

Comment:
this was `/opt/tectonic/tectonic`. So it should be `/opt/openshift/openshift` ?

Reply:
The name of the service was changed from tectonic.service to manifests.service. The last part of the path reflects the service name.

Reply:
I have changes the name of the service to `openshift.service` and the working directory to `/opt/openshift/openshift`.